cl/scrapers/tests.py

```diff
@@ -284,7 +284,7 @@ def test_press_on_with_dup_found_and_older_date(self):
                 lookup_by='sha1'
             )
             if dup_checker.full_crawl:
-                self.assertTrue(
+                self.assertFalse(
```

Comment:
I think this was correct with the `assertTrue`. The idea here is that sometimes when you're scraping you want to force the dup checker to always press on and to never report that a duplicate was found. In that case, `onwards` should always be `True`. 

I think in this case at least, the tests are working as designed and this might have surfaced a bug.